package/pom.xml

```diff
@@ -61,22 +43,21 @@
     <appframeworkVersion>2.16.0</appframeworkVersion>
 
     <!-- OpenMRS Core -->
-    <openMRSVersion>2.3.3</openMRSVersion>
-
+    <openMRSVersion>2.4.0</openMRSVersion>
```

Comment:
I am suprised that this works. Last time I tried Web Services REST didn't work with 2.4.0 🤔